ds_judgements_public_ui/sass/includes/_homepage_cards.scss

```diff
@@ -0,0 +1,146 @@
+.homepage-cards {
+  &__container {
+    padding-top: $space-8;
+    padding-bottom: 0;
+
+    h2 {
+      display: none;
+
+      @media (max-width: $grid-breakpoint-medium) {
+        @include heading_text;
+        @include header($typography-header-h2-size);
+
+        display: contents;
+        margin-left: 0;
+        text-wrap: wrap;
+      }
+    }
+
+    @media (max-width: $grid-breakpoint-medium) {
+      margin-top: $space-4;
+      margin-bottom: $space-4;
+      padding: $space-3;
+      padding-bottom: 12.5rem;
+
+      border: 3px solid $color-yellow;
+    }
+
+    @include container;
+  }
+
+  &__two_thirds_info_cards_home {
```

Comment:
I'm surprised stylelint isn't complaining about this - this isn't strictly a valid BEM class name. This should be `homepage-cards__two-thirds-info-cards-home`

After looking at where this is used and the markup though we basically have an inner container and an outer container. Probably worth naming them as such. Then the other classes could become:

```
.homepage-cards__container -> homepage-cards__outer-container
.homepage-cards__one_third_info_cards_home -> .homepage-cards__inner-container
.homepage-cards__one_third_info_cards_home_image -> .homepage-cards__image
.homepage-cards__one_third_info_cards_home_body -> .homepage-cards__body
.homepage-cards__one_third_grid_info_card_home_heading -> .homepage-cards__heading
.homepage-cards__one_third_grid_info_card_home_body_text -> .homepage-cards__body-text
```

Reply:
@jlhdxw blocked cant work on this branch see dev channel

Reply:
done please review again @jlhdxw